### PR TITLE
feat(cron): Add node cron library instrumentation docs

### PR DESCRIPTION
**Pull request by AbhiPrasad**
Documents https://github.com/getsentry/sentry-javascript/pull/9999 and https://github.com/getsentry/sentry-javascript/pull/9904

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

File: src/includes/javascript-crons-automatic-crons-instrumentation.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    console.log("You will see this message every minute");
+  },
+});
+```
+
+### Node Cron
+
+Use `Sentry.cron.instrumentNodeCron` to instrument cron export from the `node-cron` library. This returns an object with the same API as the original `cron` export, but with the `schedule` method instrumented. You can pass the name of the cron monitor and an optional timezone as part of the third options argument to the function.
+
+Requires SDK version `7.92.0` or higher.
```
Comment (shanamatthews) [inline src/includes/javascript-crons-automatic-crons-instrumentation.mdx:32]: If both options have this as a prereq, should we move this line to directly under "Automatic Crons Instrumentation"?

**Review by shanamatthews — APPROVED** [review]
A few small comments, but overall looks good!

File: src/includes/javascript-crons-automatic-crons-instrumentation.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  cronTime: "* * * * *",
+  onTick: () => {
+    console.log("You will see this message every minute");
+  },
+});
+```
+
+### Node Cron
+
+Use `Sentry.cron.instrumentNodeCron` to instrument cron export from the `node-cron` library. This returns an object with the same API as the original `cron` export, but with the `schedule` method instrumented. You can pass the name of the cron monitor and an optional timezone as part of the third options argument to the function.
```
[the remark this answers is not in the document]
Comment (AbhiPrasad) [reply src/includes/javascript-crons-automatic-crons-instrumentation.mdx:30]: I think that flows better

File: src/includes/javascript-crons-automatic-crons-instrumentation.mdx (hunk 1 above)
Comment (AbhiPrasad) [reply src/includes/javascript-crons-automatic-crons-instrumentation.mdx:32]: Yes I'll make the change, good idea.

**Pull request merged**